### DISCUSSION TITLE
Permissions issue

### DISCUSSION THREAD

**Discussion by SidhuCodes69** [vercel/next.js#48409] [Help] [answered]
### Summary

I have this error down below getting this error again and again I have deleted so many projects and download again but not working at all I am trying to give this file permissions but still nothing work :( Please help

### Additional information

```js
uncaughtException [Error: EPERM: operation not permitted, open 'C:\Users\ali\One
Drive\Desktop\NodeJS\app-portfolio\.next\trace'] {
  errno: -4048,
  code: 'EPERM',
  syscall: 'open',
  path: 'C:\\Users\\ali\\OneDrive\\Desktop\\NodeJS\\app-portfolio\\.next\\trace'
}
```

**Comment by ciruz** [comment]
Are you working in a OneDrive / Google Drive - cloud directory with sync enabled? Try to move your project out of this cloud directory. 

Why not develop with git + GitHub? 💪🏼

PS: you can safely delete .next folder, and try again. Please dont work in a cloud directory with sync enabled. Since you create and delete files a lot while in development mode or while building your project, then the OneDrive sync start syncing your files in the same time and things get difficult.

**Reply by ciruz** [reply, marked as answer]
This error can also occur if you run the dev mode in one terminal and try to build the project in another (second) terminal at the same time.

**Reply by SidhuCodes69** [reply]
That's 100% correct bro that was the issue fixed it by searching and when I turn off my NPM run dev and then run npm build it worked, and I also clean npm cache clean --force then it works that was the problem you are saying in this message thanks bro

Thanks ciruz very cool!!

**Reply by ciruz** [reply]
Great, good luck with your project
